tree-sitter-dscexpression/grammar.js

```diff
@@ -0,0 +1,25 @@
+module.exports = grammar({
+  name: 'dscexpression',
+
+  rules: {
+    statement: $ => choice(
+      $.escapedStringLiteral,
+      $.expression,
+      $.stringLiteral,
+    ),
+    escapedStringLiteral: $ => token(seq('[[', /.*/)),
+    expression: $ => seq('[', $.function, optional($._members), ']'),
+    stringLiteral: $ => token(prec(-11, /.*/)),
+    function: $ => seq($.functionName, '(', optional($._arguments), ')'),
+    functionName: $ => /[a-zA-Z]+/,
+    _arguments: $ => seq($._argument, repeat(seq(',', $._argument))),
+    _argument: $ => choice($.function, $.string, $.number, $.boolean),
```

Comment:
Can argument be a member expression? Like `[funcA(funcB().member)]`.

Reply:
yes, good catch